deploy/helm/sumologic/values.yaml

```diff
@@ -454,7 +458,7 @@ sumologic:
         ## Option to turn autoscaling on for otelcol and specify params for HPA.
         ## Autoscaling needs metrics-server to access cpu metrics.
         autoscaling:
-          enabled: false
+          # enabled: true
```

Comment:
shouldn't be `false` as well?

Reply:
To the contrary, shouldn't it be true everywhere? That's the actual default after this change.

Reply:
What's the point of uncommenting it if it's true anyway?
We are using `#` more like examples in other cases in this file